### DISCUSSION TITLE
JDBC / ODBC driver support

### DISCUSSION THREAD

**Discussion by jmls** [prisma/prisma#13131] [Q&A]
I need to access an openedge database , which is not directly supported by prisma. This server does, though, have a good jdbc / odbc driver and I was wondering if there is a generic jdbc connector that we could use to connect prisma ?

If not, how hard would it be to create one ?

**Comment by janpio** [comment]
No, unfortunately not.
And it would be rather hard, as the JDBC/ODBC ecosystem in Rust, which the Prisma Engines use, is not that great.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
